lib/ClassMover/Domain/Model/ClassMemberQuery.php

```diff
@@ -17,17 +17,10 @@ final class ClassMemberQuery
         self::TYPE_PROPERTY
     ];
 
-    private ?Class_ $class;
-
-    private ?MemberName $memberName;
-
     private ?string $type;
 
-    private function __construct(Class_ $class = null, MemberName $memberName = null, string $type = null)
+    private function __construct(private ?\Phpactor\ClassMover\Domain\Model\Class_ $class = null, private ?\Phpactor\ClassMover\Domain\Name\MemberName $memberName = null, string $type = null)
```

Comment:
I think we're missing another rector to convert these to imports? and/or possibly there is a cs fixer fix for that

Reply:
Probably the job of the CS fixer. But I didn't found the rule for that but there weren't many instances so I manually fixed them. Luckily Simplify FQN is a thing now. :D

Reply:
https://mlocati.github.io/php-cs-fixer-configurator/#version:3.13%7Cfixer:fully_qualified_strict_types :)